server/Justfile

```
@@ -13,3 +13,8 @@ fmt:
     goimports -w .
     go fmt ./...
 
+psql:
+    @PGPASSWORD=$POSTGRES_PASSWORD psql -h localhost -U $POSTGRES_USER $POSTGRES_DB
+
+create_migration name:
```

```suggestion
migration-create name:
```

If we add new commands related to migrations in the future, they will be listed together by Just

---

Good catch, thanks 🙇